### ISSUE TITLE
Linux Massive Conflicts Alfresco 5 and Cmissync 2.6.5

### ISSUE THREAD

**Issue by linock** [aegif/CmisSync#733]
Context : Alfresco Community ProductVersion: 5.0.0 (r86473-b92)
Cmissync : 2.6.5.0 for linux
Distribution : Xubuntu 16.0.4 LTS

When Cmissync is launched, at first it works but when the session is left and then opened again, a massive conflict occurs. Changing refresh timings does not solve the issue. Of course Alfresco has been modified in the global config file to prevent useless crawling and syncing accordingly to cmissync documentation.

Below the log's content

2016-11-30 10:42:51,722 [-1897617536] INFO  CmisSync.Lib.Database.DatabaseMigration [(null)] - Checking whether database /home/usr/.config/cmissync/Alfresco.cmissync exists
2016-11-30 10:43:36,509 [Threadpool worker] INFO  CmisSync.Lib.Sync.CmisRepo [(null)] - Updated sync settings. Restarting sync.
2016-11-30 10:43:36,515 [Threadpool worker] INFO  CmisSync.Lib.RepoBase [(null)] - Full Sync Started: /home/usr/CmisSync/Alfresco
2016-11-30 10:43:45,354 [Threadpool worker] INFO  CmisSync.Auth.Auth [(null)] - VendorName: Alfresco
2016-11-30 10:43:45,354 [Threadpool worker] INFO  CmisSync.Auth.Auth [(null)] - ProductName: Alfresco Community
2016-11-30 10:43:45,355 [Threadpool worker] INFO  CmisSync.Auth.Auth [(null)] - ProductVersion: 5.0.0 (r86473-b92)
2016-11-30 10:43:45,355 [Threadpool worker] INFO  CmisSync.Auth.Auth [(null)] - CmisVersionSupported: 1.1
2016-11-30 10:43:45,355 [Threadpool worker] INFO  CmisSync.Auth.Auth [(null)] - Name:
2016-11-30 10:43:45,355 [Threadpool worker] INFO  CmisSync.Auth.Auth [(null)] - Description:
2016-11-30 10:44:03,489 [Threadpool worker] WARN  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Unknown object type: 'I:act:action' (I:act:action) for object DotCMIS.Client.Impl.Folder/94164d42-fcde-40de-8b66-5934395fa9b1
2016-11-30 10:44:03,490 [Threadpool worker] WARN  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Unknown object type: 'I:act:action' (I:act:action) for object DotCMIS.Client.Impl.Folder/c672ce8a-05c0-48bb-aae8-54c944eaf6cf
2016-11-30 10:44:03,490 [Threadpool worker] WARN  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Unknown object type: 'I:act:action' (I:act:action) for object DotCMIS.Client.Impl.Folder/7d4d54cd-6778-4ce9-bbc3-d57447fe1db9
2016-11-30 10:44:03,491 [Threadpool worker] WARN  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Unknown object type: 'I:act:action' (I:act:action) for object DotCMIS.Client.Impl.Folder/90bcb173-ec66-4a2f-9d93-f584ef292d81
2016-11-30 10:44:03,491 [Threadpool worker] WARN  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Unknown object type: 'I:act:action' (I:act:action) for object DotCMIS.Client.Impl.Folder/60a01ea4-37ed-46f7-9dea-d03149bc51ab
2016-11-30 10:44:03,491 [Threadpool worker] WARN  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Unknown object type: 'I:act:action' (I:act:action) for object DotCMIS.Client.Impl.Folder/b2f0bfe2-0e00-4c38-a06f-5b008a997dcd
2016-11-30 10:44:03,491 [Threadpool worker] WARN  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Unknown object type: 'I:act:action' (I:act:action) for object DotCMIS.Client.Impl.Folder/932f3670-53e5-420a-ab80-858c918df94c
2016-11-30 10:44:03,491 [Threadpool worker] WARN  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Unknown object type: 'I:act:action' (I:act:action) for object DotCMIS.Client.Impl.Folder/f9efe8f9-85f6-49e7-8e27-55e5b21710a4
2016-11-30 10:44:16,425 [Threadpool worker] INFO  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Downloading file absent from database: /home/usr/CmisSync/Alfresco/Espaces Utilisateurs/user2/BE/ETUDES EN COURS/ CHATEAU/PHOTO VISITE/IMG_8752.JPG
2016-11-30 10:44:16,431 [Threadpool worker] INFO  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Downloading: IMG_8752.JPG
2016-11-30 10:44:17,086 [Threadpool worker] INFO  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Downloaded remote object(a58bb84d-0ad7-4d2a-9e7d-4c3bf94225e4;1.1): IMG_8752.JPG
2016-11-30 10:44:17,154 [Threadpool worker] INFO  CmisSync.Lib.Database.Database [(null)] - Checksum of /home/usr/CmisSync/Alfresco/Espaces Utilisateurs/user2/BE/ETUDES EN COURS/CHATEAU/PHOTO VISITE/IMG_8752.JPG has changed from  to 2A5337520FF637BDD831AA5F54F4DDFFB1BD37C7
2016-11-30 10:44:17,154 [Threadpool worker] INFO  CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Conflict with file: IMG_8752.JPG

**Comment by nicolas-raoul** [comment]
Does it happen all the time? Even when syncing a folder with only one file?
Thank you for the feedback!

**Comment by antton** [comment]
I wrote with the same problem here #650

Linux version is not working at least for me :(

**Comment by linock** [comment]
Ok,
so i checked everything i could and so far it seems to work well, i explain how i did : 
At first i made a full syncing from the root of the Alfresco repository, it ran all night long, we have many many files and nearly 300 users, my pc gathered all it could but it seems that cmissync was frozen for reason i ignore. In fact it was stucked with some files it was not able to handle or it was the reason invoked in the log. As the file name was looking like something not human readable, i had no clue on where it was stucked. So i killed it. 
To accelerate the tests i shrinked the number of folders to syncronize.
I explicitely checked only the shared documents root and my personal folder.
It works like a charm. So my bad probably as the first time i left all the thing to work out of the box with the root including each and every folder including all the folders belonging to Alfresco.
In short : 
Specify and add a folder to be synced  as many times as necessary and of course modifiy the local folder name where the files from Alfresco will be syncronized instead of taking the whole things in Alfresco's repository just by using -default- .
Thank you Nicolas for your kind help.
